src/Mod/Arch/ArchProfile.py

```diff
@@ -59,7 +59,7 @@ def readPresets():
     for profilefile in profilefiles:
         if os.path.exists(profilefile):
             try:
-                with open(profilefile, 'rb') as csvfile:
+                with open(profilefile) as csvfile:
```

Comment:
Hmm. I remember anything wasn't working with "rb". I think it was with the tests in python3. But this was some time ago ...
Maybe we should have  a eye on this.